src/cljs/owlet/events/contentful.cljs

```diff
@@ -44,8 +47,7 @@
                                branches :branches}]]
     (let [route-dispatch (second route-args)
           route-param (get route-args 2)
-          activities (map #(update-in % [:tag-set] (partial (comp set map) keyword))
-                        activities)
+          activities activities
```

Comment:
We can probably remove this binding all together, it should inherit from the parent scope.